wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -20,7 +20,8 @@
         <h2>Contents</h2>
         <nav>
             <ul class="unlist">
-                <li><a href="#palette">Colour palette</a></li>
+                <li><a href="#palette">Color palette</a></li>
```

Comment:
Not sure if the spelling should be changed on the page. "Color" is only used in the United States, while "colour" is used in all other variants of English.

That said, all the variable names use the spelling "color." Hm.

Reply:
Variable names (and any other developer-facing identifiers) can definitely stay as `color` - that way we're not having to juggle between the two spellings within CSS files, e.g. `background-color: $colour-red;`.

For user-facing text like the headings here, I think it's best to standardise on British English, since that's going to be what most of the existing text and documentation uses (intentionally or not). If anyone feels particularly strongly about it, the ideal thing to do would be to create an `en-US` localisation as we have for other languages, but I dare say that's not really high in anyone's priorities :-)